internal/blsgen/lib.go

```diff
@@ -9,13 +9,20 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"log"
 	"os"
+	"path/filepath"
 	"time"
 
 	ffi_bls "github.com/harmony-one/bls/ffi/go/bls"
 	"github.com/harmony-one/harmony/crypto/bls"
 )
 
+// Constants for bls lib.
+const (
```

Comment:
it can't be a constant. user should have the freedom to specify the location of the key.